Lib/ast.py

```diff
@@ -494,11 +494,14 @@ def _getter(self):
 def _setter(self, value):
     self.value = value
 
-Constant.n = property(_getter, _setter)
-Constant.s = property(_getter, _setter)
+Constant.n = property(_getter, _setter, doc='Deprecated. Use value instead.')
+Constant.s = property(_getter, _setter, doc='Deprecated. Use value instead.')
```

Comment:
Would it be possible to emit a DeprecationWarning when it's used?

Reply:
Yes, it is possible. But it would be better to start emitting warnings when the user code no longer need to use the deprecated feature. I.e. when the support of Python 3.7 be dropped in most projects which use AST.